Makefile

```diff
@@ -0,0 +1,75 @@
+BASE_IMG_NAME = bedrock_base
+DEV_IMG_NAME = bedrock_dev
+FINAL_IMG_NAME = bedrock_dev_final
+
+default:
+	@echo "You need to specify a subcommand."
+	@exit 1
+
+help:
+	@echo "build           - build docker containers for dev"
+	@echo "run             - docker-compose up the entire system for dev"
+	@echo ""
+	@echo "clean           - remove all build, test, coverage and Python artifacts"
```

Comment:
If it makes sense to your workflow not to remove the images, go ahead keep it that way. What I was looking for with my comment was to question whether you actually want to remove all artifacts -which should include docker images since it's a build artifact- /or/ remove all but docker images.